website/siteConfig.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+// List of projects/orgs using your project for the users page.
+const users = [
+  {
+    caption: 'User1',
```

Comment:
Having this user here makes the page seem broken - can we remove or comment out?

Reply:
Refer to https://github.com/Microsoft/fast-dna/pull/1175#discussion_r237646930